app/src/main/java/com/dessalines/thumbkey/utils/Utils.kt

```diff
@@ -718,7 +718,7 @@ fun getKeyboardMode(
     ) {
         KeyboardMode.NUMERIC
     } else {
-        if (autoCapitalize && autoCapitalizeCheck(ime)) {
+        if (autoCapitalize && isPasswordField(ime) && autoCapitalizeCheck(ime)) {
```

Comment:
Should be `!isPasswordField` , we don't want shifted mode if this is a password field.

Reply:
If I change it to that, it not only doesn't work for me, it also disables autocapitalization everywhere, please try testing this on your end, I think you may have something wrong with this suggestion.

Reply:
Its alright, I'll get it.